src/basic/exit-status.c

```diff
@@ -137,8 +137,8 @@ const char* exit_status_to_string(int status, ExitStatusLevel level) {
                 case EXIT_ADDRESS_FAMILIES:
                         return "ADDRESS_FAMILIES";
 
-                case EXIT_RUNTIME_DIRECTORY:
-                        return "RUNTIME_DIRECTORY";
+                case EXIT_EXEC_DIRECTORIES:
+                        return "EXEC_DIRECTORIES";
```

Comment:
I don't think we should rename the exit code here (they are API in some weak way), and instead just add four new exit codes here for the specific dirs.